extra/2-piping.js

```diff
@@ -16,26 +16,28 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(a, b) {
+  return a + b;
 }
 
-function multiply() {
-
+function multiply(a, b) {
+  return a * b;
 }
 
-function format() {
-
+function format(num) {
+  return "£" + num;
 }
 
-const startingValue = 2;
+// const startingValue = (2 + 10) * 2;
+const startingValue = (2 + 10) * 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode =
+// let badCode = format(startingValue);
+let badCode = format(startingValue);
 
 /* BETTER PRACTICE */
 
-let goodCode =
+let goodCode = format(startingValue);
```

Comment:
You should try and call the 3 functions that you have created here, as the good code example.